src/endpoints/vectors.js

```diff
@@ -109,19 +110,95 @@ async function getBatchVector(source, sourceSettings, texts, isQuery, directorie
     return results;
 }
 
+/**
+ * Extracts settings for the vectorization sources from the HTTP request headers.
+ * @param {string} source - Which source to extract settings for.
+ * @param {object} request - The HTTP request object.
+ * @returns {object} - An object that can be used as `sourceSettings` in functions that take that parameter.
+ */
+function getSourceSettings(source, request) {
+    switch (source) {
+        case 'togetherai':
+            return {
+                model: String(request.headers['x-togetherai-model']),
```

Comment:
I know this is just copied from the old place. But why is this lowercase headers, while they are set as case sensitive?

(Also, unrelated, but I learned due to tons of research at work that using "X-" prefix for headers is [not recommended anymore since like 2011](https://stackoverflow.com/a/3561399/2505580))